magedeungaro/heres-my-pet
# implement file service tests

- configured factory bot and rspec w/ capybara; factory models for user and pet; deleted test folder; first pet system test successful
- some rspec configs
- first service test
- test if FileService creates a png file if no extension is given
- style fix in fileservice rb
- testing if FileService.create_tmp_file will raise an error if no argument is given
- added positional parameter to pu.sh script to use flag
- testing script to run just a part/folder
- deleted test for FileService.create_tmp_file. Wrong testing.
- change parameter name to be more semantic (file to file_path in FileService.create_bÃlob
- change parameter name to be more semantic (file to file_path in FileService.create_bÃlob
- changed file to file_content to be more semantic
- changed name to file_name to be more semantic
- create_blob should return ActiveStorage::Blob object test
- changed 'create_after_upload!' to 'create_and_upload!' - deprecated
- changed file to file_path in FileService tests to be more semantic
- delete files after testing FileService methods
- removed a test txt file
- edited the tests.sh script to loop through spec folder and run tests
- edited the tests.sh script to loop through spec folder and run tests
- deleted channels test file
- amend.sh script so I don't forget to pull next time I amend a commit...
- amend.sh script so I don't forget to pull next time I amend a commit...
- deleted the scriptin file
- styling fix in model pet.rb
- refactored method call on model file to meet the refactored code on FileService
- refactored test to explicitly pass a file extension
- Test: .attach_file should attach the file for a given model on a given attachable
- Test: .attach_file It should not raise an error if the attachable given is not found in the model
- refactored script to be cleaner and more performatic
- fixed test to pass an attachable type not found in the model
- model creation refactored to be inside a block not a context
- Test: FileService.purge_attachment should purge the file for a given model on a given attachable
- Test: FileService.purge_attachment should not raise an error if the attachable given is not found in the model
